codeReview/cr2/Practices/separation practice/EvenRandom.cpp

```diff
@@ -0,0 +1,24 @@
+#include "EvenRandom.h"
+
+EvenRandom::EvenRandom()
+{
+	srand((unsigned)time(0));
+}
+
+int EvenRandom::ERandom()
+{
+	int rNum = rand();
+	if (rNum % 2 == 0)
+		return rNum;
+	else
+		return -1;
```

Comment:
Перевірка дублюється, можна перенести у інший метод
```suggestion
randomNumberEven(){
int randomNumber = generateRandomNumber();
if(isNumberEven(randomNumber)){
...
}
```